packages/promise-kit/src/promiseKit.js

```diff
@@ -26,7 +27,7 @@ const BestPipelinablePromise = globalThis.HandledPromise || Promise;
 /**
  * Needed to prevent type errors where functions are detected to be undefined.
  */
-const NOOP_INITIALIZER = harden(_ => {});
+const NOOP_INITIALIZER = harden(() => {});
```

Comment:
The `_` parameter list suggests that it does have a parameter, which we're ignoring.
The `()` suggests that it has no parameters.

The first one is accurate, right? (This criticism actually applies to a tremendous amount of our code, but getting types accurate seems an opportune time to correct it.)